src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2140,6 +2140,17 @@ void processContextElement
 
   while (cursor->more())
   {
+    if (strcasecmp(action.c_str(), "appendonly") == 0)
```

Comment:
In fact, the operation "POST /v2/entities/<entityId>" assumes that the entity already exists... (in order to create a new entity, "POST /v2/entities" is used).